api/client/login.go

```diff
@@ -26,7 +26,6 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 
 	flUser := cmd.String([]string{"u", "-username"}, "", "Username")
 	flPassword := cmd.String([]string{"p", "-password"}, "", "Password")
-	flEmail := cmd.String([]string{"e", "-email"}, "", "Email")
```

Comment:
@thaJeztah if we mark it as deprecated, does that mean, we keep it there, we just ignore the value that is passed in? I'm fine with what ever you think is best, I just want to make sure I understand what you are saying, so that I can implement it correctly.